src/language/typescript.ts

```diff
@@ -135,8 +135,10 @@ const UTILS_DIRECTORY = 'utils';
 const UTILS_FILENAME = 'utils';
 
 const getRelativeRoot = (cwd: string) => path.relative(cwd, ROOT_DIRECTORY);
-const getRelativeDefinitionPath = (cwd: string, definitionFileName: string): string =>
-	`${getRelativeRoot(cwd)}/${DEFINITIONS_DIRECTORY}/${definitionFileName}`;
+const getRelativeRefPath = (cwd: string, refBlockName: string, refFileName: string): string =>
+	`${getRelativeRoot(cwd)}/${refBlockName}/${refFileName}`;
+const getRelativeOutRefPath = (cwd: string, blockName: string, outFileName: string, refFileName: string): string =>
+	`${getRelativeRoot(cwd)}/../${outFileName}/${blockName}/${refFileName}`;
```

Comment:
why `/../`?

Reply:
because this is the way to the file

Reply:
what file? why '../'?

Reply:
this is for dependency from other file. please review previous PR first